### ISSUE TITLE
Different visualisation from Altair_saver

### ISSUE THREAD

**Issue by Amarounek** [altair-viz/altair_saver#23]
Hello,

I would like to show you a different visualization if you use Altair_saver

Here's my code
```
import altair as alt
from altair_saver import save
import numpy as np
import pandas as pd

alt.data_transformers.disable_max_rows()
condition = ['Dry', 'Wet']
RSDF9 = pd.DataFrame({
    'DAY': pd.date_range('1990-01-01', freq='D', periods=10000),
    'CONDITION_RR': np.random.choice(condition , 10000,)
})

bar3 = alt.Chart(RSDF9).mark_bar().encode(
    alt.X('monthdate(DAY):O', title='day', axis=alt.Axis(tickCount=5)),
    alt.Y('year(DAY):O', title='year'),
    alt.Color('CONDITION_RR:N',
              scale=alt.Scale(
              domain=['Dry', 'Wet'],
              range=['#d62728', '#1f77b4'])),
    order=alt.Order("CONDITION_RR", sort="ascending")
)
(bar3).properties(width=600, title="Dry and Wet season (1983 - 2019)").configure_axisX(labelAlign="right", labelAngle=-45, titlePadding=5)
save(bar3, 'development_dry_season.svg')
```
The manually downloaded image:
![image](https://user-images.githubusercontent.com/36733547/73639705-25561700-466d-11ea-8ef4-2857be033849.png)

By Altair_saver:
![image](https://user-images.githubusercontent.com/36733547/73639803-5df5f080-466d-11ea-8b0c-92774b35eee7.png)

**Comment by jakevdp** [comment]
Can you share a full code snippet that creates a chart that shows this issue? I don't know what ``RSDF9`` is.

**Comment by Amarounek** [comment]
OK

**Comment by Amarounek** [comment]
Edited

**Comment by jakevdp** [comment]
Thanks - it looks like the ``width`` argument is not being respected, and this is the case in both the selenium and nodejs renderers. I'll try to dig-in and figure out what's happening.

If you figure anything out as you investigate, please let me know.

**Comment by Amarounek** [comment]
Ok thx!

**Comment by joelostblom** [comment]
Since Altair 5.2, the functionality of Altair Saver is now available in Altair via the [vl-convert](https://github.com/vega/vl-convert) package. Most of the functionality has been available since 5.0, and [the main addition in 5.2 was PDF export](https://github.com/altair-viz/altair/pull/3244). See [the docs on how to save charts](https://altair-viz.github.io/user_guide/saving_charts.html#png-svg-and-pdf-format) for more details.

We are going to archive this repo, so I'm closing all the open issues and PRs before doing so. Try out the new options for saving charts mentioned above and if you run into issues, please open an issue directly in the altair or vl-convert repo.
